laborator/lab-12/01-one-tut/b-static/Makefile

```diff
@@ -0,0 +1,18 @@
+CFLAGS = -fno-PIC -m32 -g
+LDFLAGS = -static -no-pie -m32
+
+.DEFAULT_GOAL: all
+
+.PHONY: all clean
+
+all: name
+
+name: name.o
+	#$(CC) $(LDFLAGS) -o $@ $^
```

Comment:
Nu mai bine o lasi cu `gcc` in loc de tot carnatul de la `ld`? Mi se pare ca are mai mult impact prezenta flagului `-static` in `LDFLAGS` si implicit intr-o comanda mai scurta (`gcc`)

Reply:
@teodutu   Ai încercat să rulezi cu ld? Ți-a mers ?
Mie pe local nu-mi găsește lgcc și lgcc_eh. Nu știu dacă este problema asta și pe VM.

Reply:
În VM merge OK, am testat eu.

Reply:
@teodutu , am lăsat cu `gcc` implicit. Omisesem eu.

@gabrielmocanu , e nevoie să instalezi `gcc-multilib` și `libc6-dev-i386`. Am adăugat precizare în laborator.

Reply:
@gabrielmocanu , merge așa? ^

Reply:
Da, a mers.